rpcs3/Emu/RSX/VK/VKFormats.cpp

```diff
@@ -65,6 +65,16 @@ VkFilter get_mag_filter(rsx::texture_magnify_filter mag_filter)
 	throw EXCEPTION("Invalid mag filter (0x%x)", mag_filter);
 }
 
+VkBorderColor get_border_color(u8 color)
+{
+	switch (color)
+	{
+	case 0: return VK_BORDER_COLOR_FLOAT_OPAQUE_BLACK;
+	case 0xff: return VK_BORDER_COLOR_FLOAT_OPAQUE_WHITE;
```

Comment:
No need to close this; it should be okay and is actually better than having a one-size-fits-all. DX12 does set border color properly by the way. Just add a default case here that returns white and you're good to go.